lib/winston-logzio.js

```diff
@@ -52,4 +52,14 @@ module.exports = class LogzioWinstonTransport extends Transport {
   finish(callback) {
     this.logzioLogger.sendAndClose(callback);
   }
+
+  close() {
+    (async () => {
+      await new Promise((resolve) => {
+        this.finish(resolve);
+        this.emit('finish');
+        this.emit('close');
+      });
+    })();
+  }
```

Comment:
better not to use async... as it's an advance node feature:
```suggestion
 close() {
        this.finish(() => {
                this.emit('finish');
               this.emit('close');
        });
  }
```

Reply:
also there is a callback function here.
i would also check this in lambda to not break anything
and, update the version